docs/changelog/128925.yaml

```diff
@@ -0,0 +1,5 @@
+pr: 128925
+summary: Take `match_phrase` out of snapshot and make tech preview
```

Comment:
```suggestion
summary: ES|QL - Add `match_phrase` full text function (tech preview)
```

Reply:
I think the "ES|QL part will be taken care of by the `area:` parent heading when rel notes are generated :)